src/test/scala/wolfendale/scalacheck/regexp/GenParserSpec.scala

```diff
@@ -282,7 +237,7 @@ class GenParserSpec extends WordSpec with MustMatchers with PropertyChecks {
 
       forAll(gen) {
         case (min, max) =>
-          GenParser.parse(s"a{$min,$max}") mustEqual Range(Literal("a"), min, max)
+          GenParser.parse(s"a{$min,$max}") mustEqual Range(Literal('a'), min, max)
       }
     }
 
```

Comment:
I can't comment the test below as it hasn't changed but if you make the change to `Range` you can unignore this test, it will still fail until you change `max <- Gen.chooseNum(0, min)` to `max <- Gen.chooseNum(0, min - 1)` as `{1, 1}` for example is a valid range